mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -417,37 +417,38 @@ def _compute_moc_postprocess(config, streams, variableMap, calendar,
                 mocTop = _compute_moc(latAtlantic, nVertLevels, latCell,
                                       regionCellMask, transportZ, velArea)
                 mocAtlantic26[ncount] = np.amax(mocTop[:, indlat26])
-                ncount = ncount + 1
+                ncount += 1
 
             # Save to file
-            print '   save Atlantic MOC at 26.5N to yearly file...'
+            print '   save to yearly file...'
             ncFile = netCDF4.Dataset(outputFileTseries, mode='w')
             # create dimension
             ncFile.createDimension('Time', 0)
             # create variables
-            time = ncFile.createVariable('time', 'f4', ('Time',))
-            time.description = 'days since start of simulation'
+            Time = ncFile.createVariable('Time', 'f4', ('Time',))
```

Comment:
Above, you have already defined a variable `Time`.  This should be renamed `TimeVar` to avoid replacing `Time` from above since I think you want both.